src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -94,6 +101,18 @@ class Accordion extends Component<AccordionProps, State> {
     }
   };
 
+  public onClick = (e: MouseEvent) => {
+    const target = e.target as Element;
+    if (target.classList.contains('govuk-accordion__open-all')) {
```

Comment:
Could add a test for this here

Reply:
A test for?

Reply:
Hmm.. that was easier than I thought... adding a test to make sure that the onToggleAll event is fired when you click on it.

Reply:
done

Reply:
This has been completely replaced now